wp-content/themes/missilethreat/assets/_scss/components/_post-typography.scss

```diff
@@ -2,27 +2,19 @@
 
 .single__content {
 
-  /*----------  Paragraphs and In-line links  ----------*/
+  /*----------  Paragraphs ----------*/
 
-  > p {
+  > p,
+  .wp-block-columns p {
     @extend %font-body-long-lg;
-    margin-bottom: rem(40px);
+    margin-bottom: rem(32px);
     color: $color-text-dark-190;
-
-    a {
-      color: $color-primary-blue-200;
-      text-decoration: underline;
-
-      &:hover {
-        color: $color-primary-blue-500;
-      }
-    }
   }
 
-  /*----------  External Link  ----------*/
+  /*----------  Links  ----------*/
 
   /* stylelint-disable */
-  a[href*="//"]:not([href*="missile-threat.local"]):not([href*="localhost"]):not([href*="missilethreat"])
+  a[href*="//"]:not([href*="missile-threat.local"]):not([href*="localhost"]):not([href*="missilethreat"]):not([href*="csis-website-prod"])
```

Comment:
Where does the `csis-website-prod` come from?

Reply:
This is for the download PDF button, so it doesn't have an arrow icon after it.  Documents hosted on amazonaws were generating the arrow.